srcpkgs/fwupd/template

```diff
@@ -83,6 +70,7 @@ fwupd-devel_package() {
 		vmove "usr/lib/*.so"
 		vmove usr/share/gir-1.0
 		vmove usr/share/vala
+#		vmove usr/share/doc
```

Comment:
ok! last nit I think: the commit message should mention we are still using `gtk-doc`, but we are going to move to `gi-docgen` (assuming that's the plan), and there should be a comment in this line explaining what the deal is. Also it should look like:

`#vmove ....` instead of `#     vmove`

Reply:
Done.
Indeed, once the issue with gi-docgen is fixed, we should switch to it.
Presumably at next release, where all our other patches will be discarded as well.